axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
```

Comment:
Actually @eric-s-s , I'm not sure this is going to work. 

The reason we originally didn't just take the history but the player itself is that the Dual must keep track of the state of the original player, so that must include any other internal variables that would get changed by the original player.

I see two course of action:

- Keeping the `original_player` and finding a way to get it to pickle
- Add tests with more complex players and check that they both pass with or without this change (no matter what this should probably happen).
- Not only copy the player but copy all the attributes back and forth (least pleasing of all the options in my opinion)